src/ansys/tools/path/path.py

```diff
@@ -677,10 +677,17 @@ def _prompt_path(product: PRODUCT_TYPE) -> str:  # pragma: no cover
     return exe_loc
 
 
-def _clear_config_file() -> None:
-    """Used by tests. We can consider supporting it on the library"""
```

Comment:
migrate first, I think

Reply:
@Gryfenfer97 before clearing the current config file. If appdata only has one of the old configurations, this will do nothing. subsequent queries to get an app location will migrate and return the value that was just migrated from the old configuration.